.github/CODEOWNERS

```
@@ -20,3 +20,4 @@ CODEOWNERS @JuliaLang/github-actions
 /src/gc-stock.* @d-netto
 /src/gc-tls-common.h @d-netto
 /src/gc-tls-stock.h @d-netto
+/src/gc-* @udesou @qinsoon
```

```suggestion
/src/gc-* @d-netto @udesou @qinsoon
```

And then delete lines 15 through 22? It would make it a lot more concise.

---

Thanks @DilumAluthge. Good catch. 
@d-netto do you want to get notifications for everything?